### PR TITLE
bing speech api-Speech Recognition

### PR DESCRIPTION
**Pull request by cha0wg**
case 1. Add a new type definition.
- [ ] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [ ] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [ ] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

File: bing-speech/speech.1.0.0.d.ts [hunk 1]
```diff
@@ -0,0 +1,1384 @@
...
+				
+		/**
+		 * 
+		 */
+		abort(): void;
+				
+		/**
+		 * 
+		 */
+		interface HandleError {
```
Comment (sandersn) [inline bing-speech/speech.1.0.0.d.ts]: interfaces can't be nested inside other interfaces

File: bing-speech/speech.1.0.0.d.ts [hunk 2]
```diff
@@ -0,0 +1,1384 @@
...
+ *  LICENSE
+ *  -------
+ *  � 2015 Microsoft. All rights reserved.
+ *  This document is provided �as-is�. Information and views expressed in this document, including URL and other Internet Web site references, may change without notice.
+ *  Some examples depicted herein are provided for illustration only and are fictitious.  No real association or connection is intended or should be inferred.
+ *  This document does not provide you with any legal rights to any intellectual property in any Microsoft product. You may copy and use this document for your internal, reference purposes. This
+ *  document is confidential and proprietary to Microsoft. It is disclosed and can be used only pursuant to a non-disclosure agreement.
+ * ----------------------------------------------------------------------
+ */
+declare var Microsoft : {
```
Comment (sandersn) [inline bing-speech/speech.1.0.0.d.ts]: namespaces can't be nested inside object types. should be `declare namespace` like the others, I think

**Comment by paulvanbrenk** [comment]
@lvovl gentle ping

**Comment by RyanCavanaugh** [comment]
Closing due to lack of response. Please open a new PR if you'd like to continue with this change. Thanks!

**Pull request closed without merging**